### DISCUSSION TITLE
Format for output font data

### DISCUSSION THREAD

**Discussion by edgardreykh** [Chlumsky/msdf-atlas-gen#78] [Ideas]
First of all, thank u for this package, you have made a big and great tool.

I have a question, is it possible to make output data in FNT format (XML)? May be it is a stupid question, I have researched an information about fonts and there are little information and there are no big context in my head about this.

My target: find a convertor which making MTSDF xml font from ttf, because it is using in Pixi JS

**Comment by Chlumsky** [comment]
Sorry but I am not familiar with a "FNT" format and it is not supported by this project natively. However it should be easy to make a script that converts e.g. the JSON output to this format. One potential problem could be if texture coordinates are required to be integers.
